floris/turbine_library/nrel_5MW.yaml

```diff
@@ -29,31 +29,44 @@ operation_model: 'cosine-loss'
 ###
 # Parameters needed to evaluate the power and thrust produced by the turbine.
 power_thrust_table:
-  ### Power thrust table parameters
+  ### General parameters
```

Comment:
Just a heads up that these comments are used to automatically generate this page in the docs: https://nrel.github.io/floris/input_reference_turbine.html. The structure (number of hashes) and location relative to the key/value pairs does matter. If you haven't it's worthwhile to build the docs locally and see the result.

Reply:
Oh, my bad, thanks---I'll do that and adjust as needed